src/screen.cpp

```diff
@@ -32,7 +32,7 @@
 #endif
 
 /* Allow enforcing the GL2 implementation of NanoVG */
-#define NANOVG_GL3_IMPLEMENTATION
+//#define NANOVG_GL3_IMPLEMENTATION
```

Comment:
Better to remove that line completely then.

But I wonder: where is the actual used implementation now defined? (e.g. GL vs GLES, etc.). I assume the CMake build system has to detect and set this somehow..